services/motion/motion_test.go

```diff
@@ -1542,3 +1369,81 @@ func validMoveOnGlobeRequest() MoveOnGlobeReq {
 		Extra: nil,
 	}
 }
+
+// ------------------------------------------------------------------------------------------------------------------------------------
+// The below functions exist for round trip testing purposes but since proto plans cannot be converted into a valid motionplan.Plan
+// these cannot exist in the main
+// ------------------------------------------------------------------------------------------------------------------------------------
+
+// // planWithStatusFromProto converts a *pb.PlanWithStatus to a PlanWithStatus.
```

Comment:
Are these intended to be commented out?

Reply:
I missed this.  They should be deleted as they are duplicates